src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -7,20 +7,19 @@
 ;; Schemas
 
 (def Type (schema/enum :string :regex))
-(def Method (schema/enum :get :post :put :delete :head :any))
+(def Methods (schema/enum :get :post :put :delete :head :any))
+(def Method (schema/either Methods [Methods]))
```

Comment:
Yeah I guess my initial intention was that `Method` was the schema name for the key called `method`